gatsby/src/pages/bridges.js

```diff
@@ -35,7 +35,7 @@ const Bridges = ({ data }) => {
 ["Google Chat", "google-chat"],
 ["Mastodon", "mastodon"],
 ["Mattermost", "mattermost"],
-["libpurple", "libpurple"],
+["XMPP", "xmpp"],
```

Comment:
@JokerGermany: if you've figured out wtf this list does, could you add a comment to explain it? The whole thing is completely incomprehensible to me.

Reply:
@richvdh Haven't looked at the code driving the page but I assume it controls the grouping on https://matrix.org/bridges/

AFAICT, this is a mapping is from the `bridges` field to the URL slug like https://matrix.org/bridges/#google-chat

Reply:
thanks, though I'm still of the view that this ought to be a comment in the code and we ought to take this opportunity to fix that.

Reply:
yes the problem is that the bridge is still listed as libpurple and the bridge didn't use any libpurple anymore and some of the XMPP Community isn't happy that the XMPP Bridge isn't listed properly...